User/toby/working

tagd opened the pull request:
Pulling some changes from my current working to main

Commit:
Added versions of docs for me to customise

Commit:
Removed redundant pin

Commit:
Created function to toggle doc links in org-mode

Commit:
Moved pdfedit path to windows paths

Noticed issues when running config on linux

Commit:
Fix error on initial load of pdf-continuos-scroll

Error was thrown when pdf-contin is run before pdf-tools-install finish

This is currently prevented by loading pdf-contin at the end of the file giving the initial build time to work.  This is a terrible solution, look into pipes and threading etc.

Commit:
Added file for windows keybindings

Commit:
Changes aesthetic options

Moved more values to aesthetics and made visual-line-mode global

Commit:
Removed custom and added to gitignore

Unused and creating own changes

Commit:
Added diminish package

Allows hiding major mode names from mode list line

Commit:
Diminished global  minor modes

Weren't providing useful info so removed them

Commit:
Added hook to hide line numbers on pdf's

Line numbers were needless as each pdf page was read as a single line

Commit:
Toggled custom options

Hide scroll bar and make inactive buffers less dim

Would only change options locally if it was not just me using the repo.

TRSloth commented:
Being that edits have been made in other files it would be easier to clone a branch from this and do some dirty(non atomic) commits to make the branches easy to merge while retaining history.

Commit:
Make .gitignore compatible with main branch

Removed less restrictive file structures to keep main tidier

Commit:
Adding changes to my-paths to the example version

mv my version to the example and unstaged any specific customisations

Commit:
Added my newer config to customisations

fixed issues with line wrap, added column and row number, also removed visual fluff

mv of the changes from my-version.

Commit:
Redundant file removal

Ideally I'd rather keep these files and merge the pull ignoring them

 But I'm not sure how so this will do.

tagd reviewed (commented):
All looks good and ready to merge
